### ISSUE TITLE
Remove support email from code

### ISSUE THREAD

**Issue by Anthchirp** [dials/dials#830]
I was just reading through https://www.nature.com/articles/d41586-019-02046-0 and got stuck at

> Do not respond to private mails for support from users (..) Public forums offer three advantages. First, they reach many more users than do individual e-mails. “For everybody who writes an e-mail, there’s probably 100 people who are too shy to ask,” says Huber. Second, they tend to encourage more focused and thoughtful questions.

Should we remove the dials-support mail address from the code and replace it with a link to the github issue tracker?

**Comment by ndevenish** [comment]
Not to say that it doesn't, but I wonder about the demographics of scientists who are more likely to email versus file an issue ticket and how this applies to our user base; to me, an issue feels a little more of a formal step, not just a "discussion forum", and "ticket closed" can often feel a little aggressive (see for instance the common problems people have running out of memory).

It would give more of a public record of the issues people have, however, and I can see definite benefits, especially to people more used to this form of interaction (TBH I do find it a little offputting every time I have to go look up some mailman list for a packages support).

Also, nowadays, a sourceforge address does appear a little bit "How do you do fellow kids".

**Comment by dagewa** [comment]
My gut feeling is "no", offering up a support email address when the program crashes seems a friendly gesture that other packages don't offer. It says we care that the program works properly (assuming we reply promptly to resulting emails). Our good level of user support has been noted by others. A GitHub link seems a bit too... generic? I'm ready to be persuaded by other arguments though.

**Comment by rjgildea** [comment]
Also to consider is that sometimes the users might be limited in the detail they can post to a public forum if the program output/error, or information necessary to reproduce contains scientifically sensitive information.

**Comment by ndevenish** [comment]
> Also to consider is that sometimes the users might be limited in the detail they can post to a public forum if the program output/error, or information necessary to reproduce the error, contains scientifically sensitive information.

Yes, in many such cases the contact might be discouraged, or coordinate to move to email anyway

**Comment by graeme-winter** [comment]
Suggest:
- add an "issues" page on the wiki where we can document workarounds for common-ish issues (perhaps on a version-by-version basis)
- at the bottom of this page include
  - link to GitHub issue creation page with guidance on what to include
  - link to mailing list for "personal" service

Also means we can update the contact details once releases in the wild more flexibly.

**Comment by graeme-winter** [comment]
Folks - any further comments on this? Probably worth deciding on before 2.x version goes out...

**Comment by phyy-nx** [comment]
Also in agreement that the mailing list is better than a link to a github
issues tracker.

**Comment by graeme-winter** [comment]
I can work with that

Thanks for the input

**Comment by graeme-winter** [comment]
Given the interest in keeping the support email, should we work the issue tracker into:

```
@contextlib.contextmanager
def show_mail_on_error():
    try:
        yield
    except Exception as e:
        text = "Please report this error to dials-support@lists.sourceforge.net:"
        if len(e.args) == 0:
            e.args = (text,)
        elif issubclass(e.__class__, Sorry):
            raise
        elif len(e.args) == 1:
            e.args = (text + " " + str(e.args[0]),)
        else:
            e.args = (text,) + e.args
        raise
```

as well?

**Comment by dagewa** [comment]
I see the issue tracker as mainly being for developers (with GitHub accounts) who already know where to look, and the email address for the end user.

If we do put user level information in the wiki then that could be added to this message. Otherwise, I don't see the point.

**Comment by rjgildea** [comment]
On the subject of support emails... would there be any possibility of getting a better support email address? It seems a bit odd to have a sourceforge email address when we no longer use sourceforge at all (as well as the address itself being a bit verbose/cumbersome).

**Comment by dagewa** [comment]
I could do a jiscmail one. Possible advantage/disadvantage would be a public archive, like ccp4bb

**Comment by dagewa** [comment]
Another factor to consider is that it would have an .ac.uk domain

**Comment by graeme-winter** [comment]
So... do we want to act on this one or no? 🙂

**Comment by dagewa** [comment]
On the issue in the title, consensus seems to be no. For a better support email address => new issue

**Comment by ndevenish** [comment]
I agree, there are certainly people using dials who aren't at all familiar or comfortable using the github interface, and we don't want to exclude them.

Shall we close this?

**Comment by graeme-winter** [comment]
Closed in lieu of #925
